src/app/helptext/jails/jails-add.ts

```diff
@@ -477,7 +480,8 @@ hostid_tooltip: T('A new jail hostid, if desired. \
 <i>1a2bc345-678d-90e1-23fa-4b56c78901de</i>.'),
 
 hostid_strict_check_placeholder: T('hostid_strict_check'),
-hostid_strict_check_tooltip: T(''),
+hostid_strict_check_tooltip: T('Checks the hostid property of the jail \
+and if they are not the same as the hosts, it does not start the jail'),
```

Comment:
This changes from singular to multiple.  Break up the sentences, too:

`` Check the <i>hostid</i> property of the jail. If not the same as the host, do not start the jail. ``